### ISSUE TITLE
Generates Deprecation warnings with newer version of Ember

### ISSUE THREAD

**Issue by Floby** [wesm87/ember-datepicker#44]
This module generates nasty deprecation warning with Ember 1.13 and errors with Ember 2.0.

```
DEPRECATION: Using `Ember.HTMLBars.makeBoundHelper` is deprecated. Please refactor to using `Ember.Helper` or `Ember.Helper.helper`. [deprecation id: ember-htmlbars.make-bound-helper]
```

**Comment by chvonrohr** [comment]
:+1:

**Comment by aarkus7** [comment]
Might be my shoddy use of it, but I'm getting this deprecation:
A property of <> was modified inside the didInsertElement hook. You should never change properties on components, services or models during didInsertElement because it causes significant performance degradation. [deprecation id: ember-views.dispatching-modify-property]

**Comment by QuantumInformation** [comment]
I'm getting:
```
DEPRECATION: Addon files were detected in `/Users/nikos/src/ember/node_modules/ember-cli-datepicker/addon`, but no JavaScript preprocessors were found for `ember-cli-datepicker`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-cli-datepicker`'s `package.json`.```

**Comment by alexlafroscia** [comment]
@QuantumInformation that warning will be fixed in #55
